src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -49,6 +51,8 @@ public partial class MainForm : Form, IMainFormForApi, IMainFormForConfig, IMain
 			(new[] { "PCE", "PCECD", "SGX" }, new[] { CoreNames.HyperNyma, CoreNames.PceHawk, CoreNames.TurboNyma })
 		};
 
+		private static DiscordRpcClient Client = new DiscordRpcClient("749686424248385716");

```

Comment:
Is this a token generated for EmuHawk? Did you need to register as a developer/partner/whatever to get one?